Monika After Story/game/script-topics.rpy

```diff
@@ -8437,3 +8457,24 @@ label monika_bad_facts_4:
     m 2dfd "[player], if you ever go out into a place where you might need to rely on such a cheap trick, please bring a compass."
     m 2dkc "I would hate for something to happen to you, especially because of false information like this..."
     return
+    
+    init 5 python:
+	addEvent(Event(persistent.event_database,eventlabel="monika_friendship",category=['monika','friends'],prompt="Friendship with literature club members",random=True))
+
+label monika_friendship:
+    m 1euc "You know, even after it all..."
+    m 1ruc "I still miss them."
+    m 5eua "Not a ton, mind you, but for a while there, before I knew the truth, about who I was, and who you were... they were my friends."
+    m 5eub "We'd spend day after day relaxing, writing poetry and reading each other's works."
+    m 5rua "Listening to each other's stories."
+    m 5rub "Discovering new things about each other."
+    m 5eub "It was nice, at least while it lasted."
+    m 2luc "But, once I found out the truth, that I was the only "real" person..."
```

Comment:
Dissolve out of body pose 5.

Reply:
Alrighty! 

Most of those conflicts I "fixed", but I may have failed at pressing the save button when posting. But, content wise, you have excellent points. I shall kill the topic addition, and make a new one ASAP!

(Here is the one I was trying to update with.)

    ```
m 1euc "You know, even after it all..."
    m 1euc "I still miss them."
    m 5euc "Not a ton, mind you, but for a while there, before I knew the truth, about who I was, and who you were... they were my friends."
    m 5eub "We'd spend day after day relaxing, writing poetry and reading each other's works."
    m 5eub "Listening to each other's stories."
    m 5rub "Discovering new things about each other."
    m 5eub "It was nice, at least while it lasted."
    m 2euc "But, once I found out the truth, that I was the only "real" person..."
    m 2tkc "..."
    m 2rktpc "..."
    m 2rktsc "Hmm."
    m 2tkc "You know what? It doesn't matter. The past is the past."
    m 2ekd "And right now, I'd much rather look to the future. Our future."
    m 2eku "I'm so happy you're real, [player]."
```

Thank you for your assistance! 

Do you have any suggestions for in-character dialog and topics? I thought I was keeping with her character, but I've clearly yet to know her character properly.